api/v1_create_coin_test.go

```diff
@@ -90,14 +90,17 @@ func TestV1CreateCoin_StoresLinks(t *testing.T) {
 	requestBodyBytes, err := json.Marshal(requestBody)
 	assert.NoError(t, err)
 
-	status, _ := testPostWithWallet(t, app, "/v1/coins?user_id="+trashid.MustEncodeHashID(2), "0xc3d1d41e6872ffbd15c473d14fc3a9250be5b5e0", requestBodyBytes, map[string]string{
+	status, body := testPostWithWallet(t, app, "/v1/coins?user_id="+trashid.MustEncodeHashID(2), "0xc3d1d41e6872ffbd15c473d14fc3a9250be5b5e0", requestBodyBytes, map[string]string{
 		"Content-Type": "application/json",
 	})
 
+	if status != 201 {
+		t.Logf("Request failed with status %d, body: %s", status, string(body))
+	}
 	assert.Equal(t, 201, status)
 
 	var dbLink1, dbLink2, dbLink3, dbLink4 sql.NullString
-	err = app.pool.QueryRow(context.Background(), `
+	err = app.writePool.QueryRow(context.Background(), `
```

Comment:
we typically don't query the db in tests and just confirm writes by hitting various endpoints. this is probably fine as well, though pool.replicas[0] is used above which maybe is better?